node/src/database/db_migrations/db_migrator.rs

```diff
@@ -125,16 +124,16 @@ impl DbMigratorReal {
 
     pub fn update_schema_version(
         name_of_given_table: &str,
-        transaction: &Transaction,
+        transaction: &dyn TransactionWrapper,
         update_to: usize,
     ) -> rusqlite::Result<()> {
-        transaction.execute(
-            &format!(
+        transaction
+            .prepare(&format!(
                 "UPDATE {} SET value = {} WHERE name = 'schema_version'",
                 name_of_given_table, update_to
-            ),
-            [],
-        )?;
+            ))
+            .expect("internal rusqlite error")
+            .execute([])?;
```

Comment:
What does this empty box (`[]`) do?

Reply:
it means no arguments given after a statement has been formed. If you have a statement with for example question marks then you can give the values into the vector slice (or an array)